Activities/Java/UiPath.Java.Activities.ViewModels/Resources/ActivitiesMetadataWindows.json

```diff
@@ -0,0 +1,48 @@
+{
+  "orderedCategoryDisplayNameKeys": [],

```

Comment:
```suggestion
  "activities": [],

```

I think this is intended for property level metadata when listing the properties of activity.

Reply:
I will remove it